pop_up_menu.js

```diff
@@ -0,0 +1,272 @@
+  button1.setAttribute('href', liveVersion);
+  const span1 = document.createElement('span');
+  span1.textContent = 'See Live';
+  const img1 = document.createElement('img');
+  img1.alt = 'See Live';
+  img1.src = './img/see_live_icon.png';
+  button1.appendChild(span1);
+  button1.appendChild(img1);
+  buttonsDiv.appendChild(button1);
+
+  const button2 = document.createElement('a');
+  button2.classList.add('pop_menu_btn');
+  button2.setAttribute('href', linkSource);
+  const span2 = document.createElement('span');
+  span2.textContent = 'See source';
+
+  const img2 = document.createElement('img');
+  img2.alt = 'github logo';
+  img2.src = './img/social_media_logo/github.png';
+  button2.appendChild(span2);
+  button2.appendChild(img2);
+  buttonsDiv.appendChild(button2);
+
+  content1Div.appendChild(buttonsDiv);
+  contentDiv.appendChild(content1Div);
+  section.appendChild(contentDiv);
+
+  return section;
+}
+
+function extractDataFromSections() {
+  const workLinkArr = [
+    './img/grid_img_1.png',
+    './img/grid_img_2.png',
+    './img/grid_img_3.png',
+    './img/grid_img_4.png',
+    './img/grid_img_5.png',
+    './img/grid_img_6.png',
+  ];
+
+  const sections = document.querySelectorAll('.rec_works_section');
+  const extractedData = [];
+
+  sections.forEach((section) => {
+    const data = {};
+    const name = section.querySelector('.rec_works_header');
+    const description = section.querySelector('.recent_work_desc');
+    const technologies = section.querySelectorAll('.rec_skills_list li');
+
+    // Extract details within the tag and store in the object
+    data.nameText = name.textContent;
+    data.descriptionText = description.textContent;
+    data.skills = [];
+    technologies.forEach((skill) => {
+      data.skills.push(skill.textContent);
+    });
+    data.liveVersion = 'https://rychrr.github.io/';
+    data.linkSource = 'https://github.com/rychrr/portofolio_page';
+
+    // Push the object into the array
+    extractedData.push(data);
+  });
+
+  extractedData.forEach((item, index) => {
+    const workLinkIndex = index % workLinkArr.length;
+    item.featuredImage = workLinkArr[workLinkIndex];
+  });
+
+  return extractedData;
+}
+
+// Get Array  of List
+const extractedData = extractDataFromSections();
+const popUpMenus = [];
+
+extractedData.forEach((item) => {
+  const popUpMenu = createPopUpMenu(
+    item.nameText,
+    item.skills,
+    item.featuredImage,
+    item.descriptionText,
+    item.linkSource,
+    item.liveVersion,
+  );
+
+  popUpMenus.push(popUpMenu);
+});
+
+// Function to add blur effect
+// function addBlur() {
+//   const body = document;
+//   body.style.backdropFilter = 'blur(5px)';
+//   body.style.overflow = 'hidden';
+//   body.style.width = '100%';
+//   body.style.height = '100%';
+//   body.po
+
+// }
+
+// // Function to remove blur effect
+// function removeBlur() {
+//   const body = document;
+//   body.style.backdropFilter = 'none';
+// }
```

Comment:
[OPTIONAL] Please remove unwanted comments because it makes your codebase messy if you want to check your previous code then check your GitHub commit history :slightly_smiling_face:

Reply:
Thanks for the feedback @aamirkhan2478 ,